### DISCUSSION TITLE
mpv能否实现长按某键时倍速播放，松开时恢复原速

### DISCUSSION THREAD

**Discussion by zhongfly** [hooke007/mpv_PlayKit#183] [Q&A 问答] [answered]
看到一个寻找播放器的提问中提出了这样的需求，具体参考B站的网页播放器：长按住【右箭头键 → 】的时候三倍速播放，松开就恢复正常速度
mpv的快捷键似乎做不到？

**Comment by natural-harmonia-gropius** [comment, marked as answer]
https://github.com/Natural-Harmonia-Gropius/mpv_config/blob/main/scripts/pressaction.lua
改下 bind 和 click就可以用了

~~另外有没有实现 `mp.get_key_binding` 的办法，我不太喜欢 click 单点操作在脚本里赋值~~

**Reply by zhongfly** [reply]
脚本绑定的快捷键会覆盖掉input.conf里的吗？

**Reply by natural-harmonia-gropius** [reply]
~~会~~

**Reply by natural-harmonia-gropius** [reply]
现在不会覆盖了
